src/styles/components/_selected.scss

```diff
@@ -0,0 +1,47 @@
+/* Override --oc-blue-7 only to maintain visual consistency with the Bleuclair theme */
```

Comment:
Even though --oc-blue-7 is used, some colors were not overwritten.

`#main-menu .menu-children`
`#main-menu .menu-children li a:hover`

These are already overridden to be a different color in Bleuclair, so there is no need to change them.

`.avatar-color-5`

This is the background color for the initials icon, so I thought it didn't need to be matched.